packages/vis/build.sh

```diff
@@ -6,7 +6,7 @@ TERMUX_PKG_VERSION="0.9"
 TERMUX_PKG_SRCURL=https://github.com/martanne/vis/archive/v$TERMUX_PKG_VERSION.tar.gz
 TERMUX_PKG_SHA256=bd37ffba5535e665c1e883c25ba5f4e3307569b6d392c60f3c7d5dedd2efcfca
 TERMUX_PKG_AUTO_UPDATE=true
-TERMUX_PKG_DEPENDS="liblua53, libtermkey, lua-lpeg, ncurses"
+TERMUX_PKG_DEPENDS="liblua53, libtermkey, lua53-lpeg, ncurses"
```

Comment:
Need to bump revision here as well

Reply:
Good catch.
Should I also add a `TERMUX_PKG_REPLACES`?
```suggestion
TERMUX_PKG_DEPENDS="liblua53, libtermkey, lua53-lpeg, ncurses"
TERMUX_PKG_REPLACES='lua-lpeg'
```